Makefile

```diff
@@ -10,7 +10,7 @@ COMMIT_DATE=$(shell git log --pretty='format:%cd' --date='format:%Y%m%d' -n 1)
 MOCK_CONFIG=default
 
 all:
-	@echo "make check - Runs tree static check, unittests and functional tests"
+	@echo "make check - Runs tree static check, unittests and functional tests. Some tests are only executed when AEXPECT_TIME_SENSITIVE=yes is set."
```

Comment:
I assume this is the best location for this detail and there is no document or any other location that could also be updated.

Reply:
Yeah, I was looking for a documentation and found out there is none :-/